src/backend/src/modules/ppq/ppq.controller.ts

```diff
@@ -17,4 +34,22 @@ export class PpqController {
   async postForm(@Body() body: PpqPostDTO): Promise<PpqResultRO> {
     return this.ppqService.createPpq(body);
   }
+
+  @Get('/download/:id')
```

Comment:
does this endpoint need to be more specific (ex// download/pdf) ? or are we sure that this will be the only download that we see being used for the scope of this project?

Reply:
I was hoping to keep it simple until we have a new download option in the future [Like we followed for BC gov versioning philosophy]. And was wondering to introduce params for those options 🤔 

What do you think makes more sense? Happy to change to `/download/pdf/:id` - which is specific, too!

Reply:
I'm on board then.